From 7777d1bf6b18bfcb7e043ca1169f17b03b7d21a5 Mon Sep 17 00:00:00 2001
From: Laurentiu Palcu <laurentiu.palcu@nxp.com>
Date: Tue, 6 Mar 2018 11:18:54 +0200
Subject: [PATCH 3430/5242] MLK-17671-2: drm: imx: hdp: mscale: remove delay
 at the end of mode setting

commit  d2d38e0763a05890e6c02a4e0032ef166f5d56ea from
https://source.codeaurora.org/external/imx/linux-imx.git

Since DCSS was moved to use VIDEO2_PLL clock, HDMI phy clock is not used
anymore. Hence, this delay here is not necessary. It's been added inside
DCSS driver.

Signed-off-by: Laurentiu Palcu <laurentiu.palcu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/drm/imx/hdp/imx-hdmi.c |    2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/gpu/drm/imx/hdp/imx-hdmi.c b/drivers/gpu/drm/imx/hdp/imx-hdmi.c
index bdcab75..770c476 100644
--- a/drivers/gpu/drm/imx/hdp/imx-hdmi.c
+++ b/drivers/gpu/drm/imx/hdp/imx-hdmi.c
@@ -226,8 +226,6 @@ void hdmi_mode_set_t28hpc(state_struct *state, int vic, int format, int color_de
 		DRM_ERROR("CDN_API_HDMITX_SetVic_blocking ret = %d\n", ret);
 		return;
 	}
-
-	msleep(200);
 }
 
 int hdmi_get_edid_block(void *data, u8 *buf, u32 block, size_t len)
-- 
1.7.9.5

